### PR TITLE
Update to Vaadin 14

### PR DESCRIPTION
**Pull request by enver-haase**
Rough cut.
You probably want to create a branch for Vaadin 10 LTS and change README.md accordingly.
Also both NPM mode and BOWER (compat mode) should probably both be tested and working.

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/testbench-demo/19)

### CODE REVIEW

**Comment by enver-haase** [comment]
> Thanks for the PR.
> 
> I created a branch https://github.com/vaadin/testbench-demo/tree/10.0 so please update the README.



> Thanks for the PR.
> 
> I created a branch https://github.com/vaadin/testbench-demo/tree/10.0 so please update the README.

Here you are.

File: README.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 This project is a collection of samples on how to use Vaadin TestBench.
 If you are looking for more information on TestBench, see https://vaadin.com/docs/v10/testbench/testbench-overview.html
 
-This version is for Vaadin 10.
+This version is for Vaadin 14.
+For a version for Vaadin Platform 10, see https://github.com/vaadin/testbench-demo/tree/10.0
```
Comment (alvarezguille) [inline README.md]: change `Vaadin Platform 10` to `Vaadin 10`

File: pom.xml [hunk 2]
```diff
@@ -43,8 +43,15 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <!-- TODO remove once vaadin-components-testbench is removed from components -->
```
Comment (alvarezguille) [inline pom.xml]: This shouldn't be needed in 14

Comment (enver-haase) [reply pom.xml]: true, sorry for leaving it in originally.

File: README.md (hunk 1 above)
Comment (enver-haase) [reply README.md]: okay.

**Pull request merged**